dockerfiles/init/modules/codenvy/templates/codenvy.env.erb

```diff
@@ -22,7 +22,7 @@ JPDA_ADDRESS=<%= scope.lookupvar('codenvy::codenvy_debug_port') %>
 <% if scope.lookupvar('codenvy::codenvy_debug_suspend') == "true" %>JPDA_SUSPEND=y<% end -%>
 <% end %>
 
-JAVA_HOME=/opt/jdk
+JAVA_HOME=/usr/lib/jvm/default-jvm/jre
```

Comment:
@eivantsov @skabashnyuk maybe we should change JAVA_HOME to JDK not JRE?
like `JAVA_HOME=/usr/lib/jvm/java-1.8-openjdk/bin` I know that it starts with JRE but im not sure if we will have all options with JRE with debugging and some JVM OPTS that we set-up. WDYT guys?